manifests/params.pp

```diff
@@ -11,9 +11,12 @@
   $disable_ssl               = false
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
-
-  $database                  = 'postgres'
+  
+  $puppetdb_version          = $puppetdb::globals::version
+  $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
+  $manage_pg_repo            = false
```

Comment:
It don't disagree, but as a wise man once said: "standard is better than better"